website/package.json

```diff
@@ -4,13 +4,13 @@
   "version": "0.0.1",
   "author": "HashiCorp",
   "dependencies": {
-    "@hashicorp/nextjs-scripts": "11.1.1",
+    "@hashicorp/nextjs-scripts": "^13.5.0-canary.5",
```

Comment:
I've pulled in pre-releases for now, to ensure they'll work as expected. We'll run stable releases once we've confirmed they are indeed stable and usable, and pull them into this PR before merging.